pkg/kubelet/kubelet.go

```diff
@@ -2276,32 +2277,73 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 		Message: "Pod " + message})
 }
 
+// getNodeAnyWay() must return a *api.Node which is required by RunGeneralPredicates().
+// The *api.Node is obtained as follows:
+// 1. When kubelet is in standaloneMode, manufactures *api.Node with kubelet
+// 2. Try to get *api.Node from kubelet's nodeInfo, if failed, we try 3.
+// 3. Try to get *api.Node from apiserver
+func (kl *Kubelet) getNodeAnyWay() (*api.Node, error) {
+	var node *api.Node
+	if kl.standaloneMode {
+		if n, err := kl.initialNodeStatus(); err == nil {
+			return n, nil
+		}
```

Comment:
For the standalone mode, you'd need to return anyway. You don't want to proceed to asking the apiserver.
```
if kl.standaloneMode {
   return kl.initialNodeStatus()
}
```